### PR TITLE
Extension list JSON file

### PR DESCRIPTION
**Pull request by kevinpapst**
This file contains all currently existing extensions, required for this proposal: 
https://github.com/FreshRSS/FreshRSS/pull/1708

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      "version": 0.1,
+      "author": "Marien Fressinaud",
+      "url": "https://github.com/FreshRSS/Extensions",
+      "type": "gh-subdirectory"
+    },
+    {
+      "name": "CustomJS",
+      "description": "A FreshRSS extension which give ability to create user-specific Javascript rules to apply in addition of the actual theme",
+      "version": 0.1,
+      "author": "₣rans de Jonge",
```
Comment (Frenzie) [inline extensions.json]: Is it just my phone or did you use some odd glyph instead of F?

Comment (kevinpapst) [reply extensions.json]: I double checked that yesterday as well as it looked unfamiliar to me as well ... but it comes from https://github.com/FreshRSS/Extensions/blob/master/xExtension-CustomJS/metadata.json

Comment (Frenzie) [reply extensions.json]: I… typod my own name with the French franc? O_o That's a new one.

I guess I must've accidentally used some Compose key sequence.

Comment (kevinpapst) [reply extensions.json]: I changed your name in this file, but the metada.json is still open

**Comment by Alkarex** [comment]
We should remove the TT-RSS extension. It is not functional as it is. I have started an upgrade https://github.com/FreshRSS/FreshRSS/pull/1526 but it requires some database changes, which I plan to do for FreshRSS 1.10.

**Comment by kevinpapst** [comment]
@Alkarex removed

**Pull request merged**